examples/aws-iot-sdk-cpp/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-aws-iot-sdk-cpp)
+
+hunter_add_package(aws-iot-sdk-cpp)
+find_library(
```

Comment:
Project should generate `*Config.cmake` instead so users can do `find_package(* CONFIG REQUIRED)`.